### PR TITLE
Fixed incorrect color use on python class and function keywords

### PR DESCRIPTION
**Pull request by adambullmer**
This change make the python class/def words highlight in blue and not green. Makes the syntax more consistent with the other variants in this repo

### CODE REVIEW

**Comment by SuperPaintman** [comment]
I'm not sure if this is required. Currently `AAAPackageDev` safely converts these JSON files.

**Comment by adambullmer** [comment]
If you're not a fan of the extra diffs, I'd be more than happy to keep my changes to a minimum.

I only did it as I saw commit 9a99355f6561c83a566b701ca30f41806c55b098 apply _way_ more changes to the `Monokai Extended.tmTheme` than was done in the corresponding `JSON` file.

Looks like when JSON changes were made a while ago, the tool wasn't run and therefore the plist wasn't changed for a couple years. This that commit happened and applied some of those changes from way back unintentionally.

As far as the changes not being necessary, there was definitely an error in the JSON syntax here:
https://github.com/jonschlinkert/sublime-monokai-extended/pull/57/files?w=0#diff-61ce32f5c345807555a8958f9440895dR407
Because of that I wanted to be extra sure that the rest of the files didn't have copy-pasta errors in them either and just commited up the semi minimal changes anyways

Tl;DR
Something cropped up in a recent commit that made me think it was a good idea just to be sure all the plist files were up to date.

**Comment by SuperPaintman** [comment]
@adambullmer thanks. I fixed this, https://github.com/jonschlinkert/sublime-monokai-extended/commit/88d0558ee8eff3a8fb53711c65e23866de1bf82b#diff-61ce32f5c345807555a8958f9440895dL407

**Comment by adambullmer** [comment]
Corrected PR. Now only changes color.

**Comment by adambullmer** [comment]
Any update on this?

**Comment by jonschlinkert** [comment]
sorry for the delay!

**Comment by adambullmer** [comment]
No problem. Thanks for the merge!
